### PR TITLE
(feature) - Add support for custom DID numbers page url

### PR DESCRIPTION
**Pull request by horymury**
Added possibility to override the default DID number page url through branding data.

@saghul @hristoterezov

### CODE REVIEW

File: react/features/dynamic-branding/reducer.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ const DEFAULT_STATE = {
      */
     defaultBranding: true,
 
+    /**
+     * Url for a custom page for DID numbers list.
+     *
+     * @public
+     * @type {string}
+     */
+    didPageUrl: '',
```
Comment (saghul) [inline react/features/dynamic-branding/reducer.js:67]: Why not have it as undefined?

File: react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js [hunk 2]
```diff
@@ -42,6 +35,11 @@ type Props = {
      */
     _embedMeetingVisible: boolean,
 
+    /**
+     * The meeting invite text.
+     */
+    _invite: string,
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js:41]: `_invitationText` ?

File: react/features/invite/components/add-people-dialog/web/DialInSection.js [hunk 3]
```diff
@@ -72,4 +62,24 @@ function DialInSection({
...
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+  *     _dialIn: Object,
+  *     _dialInfoPageUrl: string,
+  * }}
+  */
+function _mapStateToProps(state) {
+
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/DialInSection.js:78]: Extra newline? Remove it please.

File: react/features/invite/components/add-people-dialog/web/DialInSection.js [hunk 4]
```diff
@@ -72,4 +62,24 @@ function DialInSection({
...
 
-export default translate(DialInSection);
+
+/**
+ * Maps (parts of) the Redux state to the associated props for the
+ * {@code DialInLink} component.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/DialInSection.js:72]: just say `@returns {Props}` here

File: react/features/no-audio-signal/components/DialInLink.js [hunk 5]
```diff
@@ -12,20 +12,14 @@ import { getDialInfoPageURL, shouldDisplayDialIn } from '../../invite';
...
-    _locationURL: string,
-
+    _dialIn: Object,
 
     /**
-     * The redux state representing the dial-in numbers feature.
+     * The url of the page containing the dial-in numbers list.
      */
-    _dialIn: Object,
+    _dialInfoPageUrl: String,
```
Comment (saghul) [inline react/features/no-audio-signal/components/DialInLink.js:22]:
```suggestion
    _dialInfoPageUrl: string,
```

File: react/features/no-audio-signal/components/DialInLink.js [hunk 6]
```diff
@@ -78,17 +67,15 @@ class DialInLink extends Component<Props> {
  * @param {Object} state - The Redux state.
  * @private
  * @returns {{
-    *     _room: string,
-    *     _locationURL: string,
     *     _dialIn: Object,
+    *     _dialInfoPageUrl: string,
```
Comment (saghul) [inline react/features/no-audio-signal/components/DialInLink.js:71]: Same as in the other file

**Review by saghul — COMMENTED** [review]
LEft some minor comments, but LGTM overall!

File: react/features/dynamic-branding/reducer.js (hunk 1 above)
Comment (horymury) [reply react/features/dynamic-branding/reducer.js:67]: I went with the same pattern with the rest of the URL strings there. Shall I change everywhere?

Comment (saghul) [reply react/features/dynamic-branding/reducer.js:67]: Consistency is better, let's keep it!

**Review by saghul — APPROVED** [review]

**Pull request merged**